site/docs/foundations/data-visualization/chart-accessibility.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Chart accessibility
+layout: DetailTechnical
+sidebar:
+  priority: 5
+---
+
+Charts rely on visual elements to make complex data easier to understand, so it’s essential to ensure they are accessible to all users. Use color appropriately, include visual cues, provide text for assistive technologies, and ensure any interactions are accessible.
+
+## Categorical colors
+
+### Color contrast
+
+Users with visual impairments should be able to fully understand data that is represented using different colors. The use of custom colors or hex codes is discouraged, as they may not meet non-text contrast requirements and can compromise accessibility. Salt tokens should be used to ensure compliance, as they are designed to achieve at least a 3:1 contrast ratio between the background and non-text chart elements such as bars, lines, segments, and icons.
+
+<Image
+  src="/img/foundations/dataviz-hex.png"
+  srcDark="/img/foundations/dataviz-hex-dark.png"
+  alt="Diagram of best practices using categorical color versus color picking."
+/>
+
+### Color order
+
+Salt's categorical colors are intended to be used in numerical order, from 1 through 20.
+
+<Image
+  src="/img/foundations/dataviz-color-order.png"
+  srcDark="/img/foundations/dataviz-color-order-dark.png"
+  alt="Diagram of the categorical color palette."
+/>
+
+A well-defined, distinct color palette ensures that everyone can easily interpret the data. Using the set categorical palette colors minimizes the risk of misinterpretation and ensures a visual order. Don’t apply similar categorical color colors to the same chart.
```

Comment:
Could this paragraph and related image be better placed on the color page. A11y requires more than just color. Recommendations about visual order (through color alone) is possibly not a strong a11y principle, but more related to the chart color principles we have provided instead.